docs/docker-machine-drivers.md

```diff
@@ -1,18 +1,23 @@
-# Driver plugin installation
+# Docker Machine driver installation
```

Comment:
@LalatenduMohanty that's what I do, right? I take this as a +1 for what I wrote or did you mean something else?